design-system/src/design-tokens.ts

```diff
@@ -1,38 +1,295 @@
+    borderColorForInputWhenReadonly: '#ccc',
+    borderColorForInputWhenError: '#e60050',
+    borderColorForInputWhenWarning: '#f16d0e',
+    borderColorForTag: 'hsl(0, 0%, 60%)',
+    borderColorForTagWarning: '#f16d0e',
+    borderColorForTagWhenFocused: '#00b39e',
+    borderRadiusForInput: '6px',
+    borderRadiusForTag: '2px',
+    fontColorForInput: '#1a1a1a',
+    fontColorForInputWhenDisabled: 'hsl(0, 0%, 60%)',
+    fontColorForInputWhenError: '#e60050',
+    fontColorForInputWhenReadonly: 'hsl(0, 0%, 60%)',
+    fontColorForInputWhenWarning: '#f16d0e',
+    fontColorForTag: '#1a1a1a',
+    fontColorForTagWhenDisabled: 'hsl(0, 0%, 60%)',
+    fontColorForTextWhenInverted: '#fff',
+    fontFamily: "'Open Sans', sans-serif",
+    placeholderFontColorForInput: 'hsl(0, 0%, 60%)',
+    fontSizeForInput: '1rem',
+    shadowBoxTagWhenHovered:
+      '0 1px 3px rgba(0, 0, 0, 0.12), 0 1px 2px rgba(0, 0, 0, 0.24)',
+    bigButtonHeight: '32px',
+    smallButtonHeight: '24px',
+    sizeHeightInput: '32px',
+    fontFamilyDefault: "'Open Sans', sans-serif",
+    fontSizeDefault: '1rem',
+    fontSizeSmall: '0.9231rem',
+    shadowBoxTagHover:
+      '0 1px 3px rgba(0, 0, 0, 0.12), 0 1px 2px rgba(0, 0, 0, 0.24)',
+    sizeHeightTag: '26px',
+    standardInputHeight: '32px',
+  },
+  dark: {
+    colorPrimary: '#f78d26',
+    colorSolid: '#fff',
+    colorSurface: '#1a1a1a',
+    backgroundColorForTag: 'hsl(25.110132158590307, 89.0196078431%, 95%)',
+    backgroundColorForTagWarning: 'hsl(0, 0%, 95%)',
+  },
+} as const;
 
-export default {
-  backgroundColorForInput: 'colorSurface',
-  backgroundColorForInputWhenSelected: 'colorAccent95',
-  backgroundColorForInputWhenDisabled: 'colorAccent98',
-  backgroundColorForInputWhenHovered: 'colorNeutral90',
-  backgroundColorForTag: 'colorNeutral95',
-  backgroundColorForTagWarning: 'colorWarning95',
-  borderColorForInput: 'colorNeutral60',
-  borderColorForInputWhenFocused: 'colorPrimary',
-  borderColorForInputWhenDisabled: 'colorNeutral',
-  borderColorForInputWhenReadonly: 'colorNeutral',
-  borderColorForInputWhenError: 'colorError',
-  borderColorForInputWhenWarning: 'colorWarning',
-  borderColorForTag: 'colorNeutral60',
-  borderColorForTagWarning: 'colorWarning',
-  borderColorForTagWhenFocused: 'colorPrimary',
-  borderRadiusForInput: 'borderRadius6',
-  borderRadiusForTag: 'borderRadius2',
-  fontColorForInput: 'colorSolid',
-  fontColorForInputWhenDisabled: 'colorNeutral60',
-  fontColorForInputWhenError: 'colorError',
-  fontColorForInputWhenReadonly: 'colorNeutral60',
-  fontColorForInputWhenWarning: 'colorWarning',
-  fontColorForTag: 'colorSolid',
-  fontColorForTagWhenDisabled: 'colorNeutral60',
-  fontColorForTextWhenInverted: 'colorSurface',
-  fontFamily: 'fontFamilyBody',
-  placeholderFontColorForInput: 'colorNeutral60',
-  fontSizeForInput: 'fontSizeM',
-  shadowBoxTagWhenHovered: 'shadow1',
+export const themesNames = {
+  default: 'default',
+  dark: 'dark',
```

Comment:
Do we want to export these values for now? I want to avoid possible confusion if these values are available as exports 🤔

Reply:
Updated in this commit: 67e20d87b162991132e047c2cd58ca00c96ed93e